docs-source/content/userguide/managing-domains/domain-lifecycle/startup.md

```diff
@@ -82,6 +83,75 @@ updates before advancing the server to the running state.
 
 Changes to the `serverStartState` property do not affect already started servers.
 
+### Domain lifecycle sample scripts
+Beginning with operator version 3.1.0, the WebLogic Server Kubernetes Operator project provides a set of sample scripts to shut-down or start-up a specific managed-server, cluster or the entire domain. These scripts are located in the `kubernetes/samples/scripts/domain-lifecycle` directory. These can be helpful when scripting the lifecycle of a WebLogic Server Domain.
+
```

Comment:
Everything remaining below should be removed - since it's now in the sample doc which you reference ^^^.

Reply:
Please add a link to the sample doc ^^^.

Reply:
LGTM